packages/governance/src/contractHelper.js

```diff
@@ -4,14 +4,30 @@ import { Far } from '@endo/marshal';
 import { makeStoredPublisherKit } from '@agoric/notifier';
 import { getMethodNames, objectMap } from '@agoric/internal';
 import { ignoreContext } from '@agoric/vat-data';
-import { keyEQ } from '@agoric/store';
+import { keyEQ, M } from '@agoric/store';
+import { AmountShape, BrandShape } from '@agoric/ertp';
 import { assertElectorateMatches } from './contractGovernance/paramManager.js';
 import { makeParamManagerFromTerms } from './contractGovernance/typedParamManager.js';
 
 const { details: X, quote: q } = assert;
 
 export const GOVERNANCE_STORAGE_KEY = 'governance';
 
+const extendPublicAPI = {
+  getSubscription: M.call().returns(M.remotable('StoredSubscription')),
+  getContractGovernor: M.call().returns(M.remotable('Instance')),
+  getGovernedParams: M.call().returns(M.record()),
+  getAmount: M.call().returns(AmountShape),
+  getBrand: M.call().returns(BrandShape),
+  getInstance: M.call().returns(M.remotable('Instance')),
+  getInstallation: M.call().returns(M.remotable('Installation')),
+  getInvitationAmount: M.call().returns(M.promise()),
+  getNat: M.call().returns(M.bigint()),
+  getRatio: M.call().returns(M.record()),
+  getString: M.call().returns(M.string()),
+  getUnknown: M.call().returns(M.string()),
```

Comment:
`returns(M.any()),`

Reply:
I'm confused. Are you suggesting that these should have broader types rather than the specifics?

Reply:
No, I'm saying that exactly `getUnknown` returns `M.any()` rather than a string. The whole point is that it's an escape from the type mechanism for types we don't support yet.

Reply:
By the notion [above about hints](https://github.com/Agoric/agoric-sdk/pull/6119#discussion_r962320291), should this say that it returns `M.unknown()`?

Reply:
How does `M.unknown()` differ from `M.any()`? Are they operationally equivalent, but if generating TS type text from patterns, would differ in what types they generate?

Reply:
Exactly. At runtime they both mean "nothing to check". As a param, any and unknown both mean "accepts without checking". At use, unknown means no guarantees (so you have to check! and any means treat it as any type.

Wondering if something like "M.defined" would be useful to guarantee that's is some value, not null or undefined. Then the caller can safely discriminate on properties.

None of these hunting ideas seem like blockers to me.

Reply:
It's easy enough to add a clone of `any` named `unknown` to allow the right definitions.

Reply:
Regardless of what we'll eventually do, getUnknown can't return `M.string()`. Please choose something else for the interim.

Reply:
Already changed to M.any.